data_processors/pipeline/orchestration/umccrise_step.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+"""dragen_tso_ctdna_step module
+
+See domain package __init__.py doc string.
+See orchestration package __init__.py doc string.
+"""
+import json
+import logging
+from typing import List, Dict
+
+from data_portal.models.workflow import Workflow
+from data_portal.models.labmetadata import  LabMetadata
+from data_processors.pipeline.domain.batch import Batcher
+from data_processors.pipeline.domain.config import SQS_UMCCRISE_QUEUE_ARN
+from data_processors.pipeline.domain.workflow import WorkflowType
+from data_processors.pipeline.services import batch_srv, fastq_srv, metadata_srv
+from data_processors.pipeline.tools import liborca
+from utils import libssm, libsqs
+
+# GLOBALS
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def perform(this_workflow: Workflow):
+
+    batcher = Batcher(
```

Comment:
So, we wish to wrap and run in Batch manner? Like `wgs_alignment_qc`. Not an issue from my side; just want to make sure this is the intent. Because its  upstream trigger; T/N workflows are not running in Batch. They run quite distributed manner by each Subject i.e. T/N completion event arrive independently; and may be interleaved, concurrent and/or no order.

But, no worries about it yet. Let me write test case to observe the feasibility for how this pipeline flow, first. May (or may not) need to change run behaviour here.

Reply:
Oh good point, to confirm the trigger for this workflow is the completion of the T/N workflow.